client/blocks/get-apps/desktop-download-card.jsx

```diff
@@ -181,7 +199,7 @@ class DesktopDownloadCard extends Component {
 	getButtonLink( platform ) {
 		switch ( platform ) {
 			case 'MacIntel':
-				return MAC_LINK;
+				return [ MAC_INTEL_LINK, MAC_SILICON_LINK ];
```

Comment:
Good approach! It works, although sometimes we would return an array and some times a string.

Let's create a new synthetic case for the silicon case:

```suggestion
				return MAC_INTEL_LINK;
		case 'MacSilicon':
				return MAC_SILICON_LINK;
```

Or even better, let's create a new constant on the top of the file:
```
const PLATFORM_MAC_SILICON = 'MacSilicon';
```
And then that variable in the rest of the cases and calls.
```suggestion
				return MAC_INTEL_LINK;
		case PLATFORM_MAC_SILICON:
				return MAC_SILICON_LINK;
```